linode/manager
# M3-5719: Show StackScripts that use deprecated images in Account StackScripts tab

## Description

Take two of #8298

- Shows StackScripts that use deprecated Images in the **Account StackScripts** tab 
  - Previously we just filtered out StackScripts that used deprecated images and user's StackScripts would not be viewable or editable at all in Cloud Manager
  - If a **Community StackScript** does **not** use any non-deprecated images (all of the StasckScripts images are deprecated), it should be hidden in the **Community StackScript** because it is not deployable. 
- Adds a new "Deprecated Images" section to the StackScript details page containing any deprecated images the StackScript uses (show in screenshot)
 
![Screen Shot 2022-03-25 at 11 04 53 AM](https://user-images.githubusercontent.com/6440455/160146915-d2fda44c-49c5-4afa-926d-940de64f2d2f.jpg)

## How to test

- Test viewing **Account StackScripts**
  - Ensure Linode Kubernetes StackScripts do not display when you have LKE clusters deployed
  - Test viewing an Account StackScript that uses a deprecated image (Refer to internal ticket)
-  Test viewing **Community StackScripts**
   - Behavior should match production exactly
   - You should not see any StackScripts only use deprecated images
- Test Deploying a Linode with the `Deploy New Linode` button on a StackScript
- Test Deploying a Linode by going through the Linode create flow and using the StackScript tab
